Doc/reference/executionmodel.rst

```diff
@@ -151,7 +152,8 @@ a global statement, the free variable is treated as a global.
 The :keyword:`nonlocal` statement causes corresponding names to refer
 to previously bound variables in the nearest enclosing function scope.
 :exc:`SyntaxError` is raised at compile time if the given name does not
-exist in any enclosing function scope. :ref:`Type parameters <type-params>`
+exist in any enclosing function scope, or if the nearest enclosing scope 
+is the global (module) scope. :ref:`Type parameters <type-params>`
```

Comment:
I think this is redundant with the previous statement that the error is raised "if the given name does not
exist in any enclosing function scope". If there is no enclosing function scope, then logically the name doesn't exist in any enclosing function scope. The new statement is also incomplete, because it's also possible that there's a class scope in between.

Reply:
Reverted this change.